### ISSUE TITLE
Add vibration customization

### ISSUE THREAD

**Issue by vertigo220** [kamaravichow/safe-dot-android#40]
First I just want to say this is a really neat little app which is very well done. I ran across it searching for something else and am glad I did, as it's the next best thing to a camera shutter.

The one thing I've found that could use some improvement is the vibration when an app uses the camera or mic. It's extremely short and weak, so it's unlikely I'd ever notice it, as I barely did even when testing it. It would be much better if it could be customized, at least duration and intensity, but possibly pattern as well.

**Comment by kamaravichow** [comment]
Yeah that makes sense, I will be adding vibration customisation soon.

Thanks for reporting @vertigo220

**Comment by vertigo220** [comment]
bump due to stale bot
